packages/apollo-server-client/index.js

```diff
@@ -0,0 +1,242 @@
+const { ApolloClient, InMemoryCache, ApolloLink } = require('@apollo/client')
+const { BatchHttpLink } = require('@apollo/client/link/batch-http')
+const { createUploadLink } = require('apollo-upload-client')
+const { onError }  = require('apollo-link-error')
+const { RetryLink } = require('@apollo/client/link/retry')
+const { getLogger } = require('@condo/keystone/logging')
+const fetch = require('cross-fetch/polyfill').fetch
+const { chunk: splitArray } = require('lodash')
+
+const { MAX_REQUESTS_IN_BATCH, MAX_MODIFY_OPERATIONS_IN_REQUEST, MAX_RETRIES_ON_NETWORK_ERROR, LOAD_CHUNK_SIZE } = require('./constants')
+const { SIGNIN_BY_EMAIL_MUTATION, SIGNIN_BY_PHONE_AND_PASSWORD_MUTATION } = require('./lib/gql')
+
+class ApolloServerClient {
+
+    client
+    clientName
+    authToken
+    userId
+    locale
+    endpoint
+    authRequisites = {}
+    logger
+
+    /**
+     *
+     * @param endpoint - https://condo.d.doma.ai/admin/api
```

Comment:
Jsdoc allows you to determine arg types)
```suggestion
     * @param {string} endpoint - https://condo.d.doma.ai/admin/api
```